physx/src/scene.rs

```diff
@@ -88,6 +91,31 @@ impl Scene {
         })
     }
 
+    pub fn add_capsule_controller(
+        &mut self,
+        material: *mut PxMaterial,
+        height: f32,
+        radius: f32,
+        step_offset: f32,
+    ) -> Result<Controller, String> {
```

Comment:
If this function takes the description instead, we don't have to return an error here. :)

Reply:
Unfortunately, it still needs one for the case when no controller manager has been added.